jenkins-pipelines/longevity-alternator-3h-multidc.jenkinsfile

```diff
@@ -9,6 +9,6 @@ longevityPipeline(
     test_name: 'longevity_test.LongevityTest.test_custom_time',
     test_config: 'test-cases/longevity/longevity-alternator-3h-multidc.yaml',
 
-    timeout: [time: 400, unit: 'MINUTES'],
+    timeout: [time: 580, unit: 'MINUTES'],
```

Comment:
remove the line completely, it is not used any more ( we use test_duration from the YAML)

Reply:
Done